src/Form/FormAnalyzer.js

```diff
@@ -23,28 +23,40 @@ class FormAnalyzer {
     constructor (form, input, matching) {
         this.form = form
         this.matching = matching || new Matching(matchingConfiguration)
+
         /**
          * The signal is a continuum where negative values imply login and positive imply signup
          * @type {number}
          */
         this.autofillSignal = 0
         /**
-         * Collects the signals for debugging purposes
-         * @type {string[]}
+         * A hybrid form can be either a login or a signup, the site uses a single form for both
+         * @type {number}
          */
-        this.signals = []
+        this.hybridSignal = 0
 
         /**
-         * A hybrid form can be either a login or a signup, the site uses a single form for both
-         * @type {boolean}
+         * Collects the signals for debugging purposes
+         * @type {string[]}
          */
-        this.isHybrid = false
+        this.signals = []
```

Comment:
The diff is weird, but I've just created a new `hybridSignal` count so that `isHybrid` is more nuanced (see below).